tests/kafkatest/services/kafka/templates/kafka.properties

```diff
@@ -12,9 +12,10 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-# see kafka.server.KafkaConfig for additional details and defaults
 
 
+# see KafkaConfig for an additional configuration mechanism and defaults
```

Comment:
I thought that "additional details and defaults" was fine, but good to add some space between the license text and this comment (as you did).